README.md

```diff
@@ -37,6 +37,22 @@ instance Monad ((->) r) where
 
 </details>
 
+<details>
+<summary>Function as Semigroup and Monoid</summary>
+
+```haskell
+-- | @since 4.9.0.0
+instance Semigroup b => Semigroup (a -> b) where
+        f <> g = \x -> f x <> g x
+        stimes n f e = stimes n (f e)
+
+-- | @since 2.01
+instance Monoid b => Monoid (a -> b) where
+        mempty _ = mempty
```

Comment:
I’m wondering if it’s better to write `mempty = const mempty`

Reply:
This is just copy paste, same as other code under GHC Source Code